Flutter/json_to_dart/lib/models/config.dart

```diff
@@ -90,6 +90,17 @@ class ConfigSetting extends Setting<ConfigSetting> with ChangeNotifier {
       notifyListeners();
     }
   }
+
+  bool _smartNullable = false;
+  @HiveField(13)
+  bool get smartNullable => _smartNullable;
```

Comment:
bool get smartNullable => _smartNullable??false;

hive issue https://github.com/hivedb/hive/issues/625

Reply:
I think it's unnecessary, the exception is thrown from config.g.dart `..smartNullable = fields[13] as bool`, `type 'Null' is not a subtype of type 'bool' in type cast`,  I directly modified config.g.dart to solve this problem temporarily, but I still suggest adding the version number to the hive.